routing/routing_integration_tests/routing_test_tools.cpp

```diff
@@ -81,14 +83,15 @@ namespace integration
     return osrmRouter;
   }
 
-  shared_ptr<IRouter> CreatePedestrianRouter(Index & index,
-                                             storage::CountryInfoGetter const & infoGetter)
+  shared_ptr<IRouter> CreateAStarRouter(Index & index,
```

Comment:
Yes. Before doing so I considered several alternatives. In that case we have factories for production using and some wrappers for these factories for test infrastructure (see the name of the file routing/routing_integration_tests/routing_test_tools.cpp). If you think we can find a better solution in that case I suggest to discuss it by voice.